tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/CounterexampleWriter.scala

```diff
@@ -21,14 +21,16 @@ object CounterexampleWriter {
   def writeAllFormats(rootModule: TlaModule, notInvariant: NotInvariant, states: List[NextState] ): List[String] = {
     new TlaCounterexampleWriter(new PrintWriter(new FileWriter("counterexample.tla"))).write(rootModule, notInvariant, states)
     new TlcCounterexampleWriter(new PrintWriter(new FileWriter("MC.out"))).write(rootModule, notInvariant, states)
-    List("counterexample.tla", "MC.out")
+    new JsonCounterexampleWriter(new PrintWriter(new FileWriter("counterexample.json"))).write(rootModule, notInvariant, states)
+    List("counterexample.tla", "MC.out", "counterexample.json")
```

Comment:
shall we close the writers in `try...finally`

Reply:
right; implemented in 7b06d47